src/data/example.1.conf

```diff
@@ -0,0 +1,46 @@
+[settings]
```

Comment:
Why add a new example.conf instead of modifying the current one?

Reply:
example.conf and example.1.conf differ on two keys
```
diff src/data/example.1.conf src/data/example.conf 
41,46d40
< 
< #server delivery disable
< record_server_delivery_enabled=false
< 
< #local copy enabled
< record_retention_enabled=true
```
I did not modify the existing one, because I want to test the behavior where these keys are not present in a conf file (the case with existing custom configuration files). Maybe this test was good for development only and now it can be dropped? I thought this is an important case (configuration backwards compatibility) to keep an eye on with a test.

Reply:
I forgot what the example.conf was used for... Sounds valid to have a separate conf. It may be helpful to rename the two conf files in the future to clearly reflect their use.

Reply:
So, +1